ansible/playbook/README.md

```diff
@@ -108,7 +124,7 @@ The section describes how to maintain hosts in Ansible and their use. It's execu
 
 ## Create a host
 
-To create a host first the host must be added to the Ansible inventory. This is done using the following statement:
```

Comment:
I think that we should be here more didactic and explain what will happen when we execute the playbook mentioned for our project. If I remember correctly, the file `ansible/inventory/hosts.yml` needs to be created first - is it correct ? Is it documented like also how the structure should be `all: children : ...` ? how we could add additional var for an host. Dont forget to add real example please